js/dispatch_json.ts

```diff
@@ -30,7 +30,10 @@ function nextPromiseId(): number {
 }
 
 function decode(ui8: Uint8Array): JsonResponse {
-  const s = new TextDecoder().decode(ui8);
+  let s = new TextDecoder().decode(ui8);
+  // TODO: this is make-shift solution
+  const closingBracket = s.lastIndexOf("}");
+  s = s.slice(0, closingBracket + 1);
```

Comment:
This seems odd / sub-optimal. What characters are you padding with? I suggest using space characters - then you shouldn't need to slice.

Reply:
That's so obvious... Thanks!